sql/util/whotracksme_trackers.py

```diff
@@ -0,0 +1,54 @@
+"""
+This module retrieves and extracts trackers as identified by WhoTracks.me
+and appends them to the httparchive.almanac.whotracksme BigQuery table.
+"""
+
+# pylint: disable=import-error
+
+from datetime import datetime as DateTime
+import sqlite3
+
+import pandas
+import requests
+from bq_writer import write_to_bq, bigquery
+
+
+# get current year
+year = DateTime.now().year
+
+# Retrieve and extract trackers as identified by WhoTracks.me.
+# https://github.com/ghostery/whotracks.me/blob/master/blog/generating_adblocker_filters.md#loading-the-data
+tracker_db = requests.get(
+    "https://raw.githubusercontent.com/whotracksme/whotracks.me/master/whotracksme/data/assets/trackerdb.sql",
+    timeout=10,
+).text
+
+trackers_query = f"""
+    SELECT
+        '{year}-06-01' AS date,
```

Comment:
Nit. Shouldn't assume it's always going to be June used (we have used July on occasion). Maybe have a constant set to the current analysis year (`2024-06-01` now)?